<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DlgPerformanceBase</class>
 <widget class="QWidget" name="DlgPerformanceBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>256</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>CPU Usage</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="kcfg_EnableCompositing" >
            <property name="text" >
             <string>Enable &amp;transparency effects</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_EnableThreading" >
            <property name="text" >
             <string>Enable &amp;background generation</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="cpuLabel" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="pixmap" >
             <pixmap/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>21</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KButtonGroup" name="kcfg_MemoryLevel" >
     <property name="title" >
      <string>Memory Usage</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="lowRadio" >
            <property name="text" >
             <string>&amp;Low</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="normalRadio" >
            <property name="text" >
             <string>&amp;Normal (default)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="aggressiveRadio" >
            <property name="text" >
             <string>&amp;Aggressive</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="memoryLabel" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="pixmap" >
             <pixmap/>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>21</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="descLabel" >
        <property name="text" >
         <string/>
        </property>
        <property name="textFormat" >
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>DesktopIcon</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global" >kiconloader.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
